lib/puppet/provider/package/pkgin.rb

```diff
@@ -30,7 +30,7 @@ def self.prefetch(packages)
   end
 
   def self.instances
-    pkgin(:list).split("\n").map do |package|
+    pkgin("-p", :list).split("\n").map do |package|
```

Comment:
Likewise, is `-p` backwards compatible?  This wasn't a recent feature added to pkgin, right?